javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -35,7 +35,7 @@ predicate calls(DataFlow::InvokeNode invk, Function f) {
 private predicate partiallyCalls(DataFlow::AdditionalPartialInvokeNode invk, DataFlow::AnalyzedNode callback, Function f) {
   invk.isPartialArgument(callback, _, _) and
   exists (AbstractFunction callee | callee = callback.getAValue() |
-    if invk.isIndefinite("global") then
+    if callback.getAValue().isIndefinite("global") then
```

Comment:
> Optionally, I would support pulling out the logic for preventing cross-file global calls shared by calls and partiallyCalls into an auxiliary predicate (with an appropriate bindingset to enforce inlining if necessary).

After this bugfix the two aren't easy to unify anymore. `invk.isIndefinite` relies on the cached predicate `getACalleeValue` which we can't use here. Rewriting it turned out to be very costly, so I'd rather leave it like this.